### ISSUE TITLE
shoutem clone git reference is not a tree

### ISSUE THREAD

**Issue by seblau** [shoutem/cli#25]
When I run `shoutem clone <APP_ID>` I get the following npm error.

Confirmed on:
* mac
* ubuntu

with npm version 3-5 and node version 6-8

```
> node scripts/configure

Starting build for app 5648
Cleaning temp files: 6.536ms
Download configuration: 1165.814ms
Load local extensions: 27.931ms
Installing dependencies:
npm WARN deprecated babel@6.23.0: In 6.x, the babel package has been deprecated in favor of babel-cli. Check https://opencollective.com/babel to support the Babel maintainers
npm WARN deprecated babel-preset-es2015@6.24.1: We're super 😸  excited that you're trying to use ES2015 syntax, but instead of making more yearly presets 😭 , Babel now has a better preset that we recommend you use instead: npm install babel-preset-env --save-dev. preset-env without options will compile ES2015+ down to ES5 just like using all the presets together and thus is more future proof. It also allows you to target specific browsers so that Babel can do less work and you can ship native ES2015+ to user 😎 ! We are also in the process of releasing v7, so please give http://babeljs.io/blog/2017/09/12/planning-for-7.0 a read and help test it out in beta! Thanks so much for using Babel 🙏, please give us a follow on Twitter @babeljs for news on Babel, join slack.babeljs.io for discussion/development and help support the project at opencollective.com/babel
npm WARN deprecated connect@2.30.2: connect 2.x series is deprecated
npm WARN deprecated node-uuid@1.4.7: Use uuid module instead
npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
npm WARN deprecated node-uuid@1.3.3: Use uuid module instead
npm WARN deprecated lodash-node@2.4.1: This package is discontinued. Use lodash@^4.0.0.
npm ERR! code 128
npm ERR! Command failed: /usr/bin/git checkout b3e12d5343e9272bf994128dee5114eeeb35d557
npm ERR! fatal: reference is not a tree: b3e12d5343e9272bf994128dee5114eeeb35d557
npm ERR! 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/mobileci/.npm/_logs/2017-09-19T13_08_52_041Z-debug.log
{ Error: `npm install` failed with code 1
    at ChildProcess.<anonymous> (/project/HAKA/scripts/node_modules/child-process-promise/lib/index.js:132:23)
    at emitTwo (events.js:125:13)
    at ChildProcess.emit (events.js:213:7)
    at maybeClose (internal/child_process.js:897:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:208:5)
  name: 'ChildProcessError',
```

**Comment by dodsky** [comment]
I think one of the depending libraries was updated and introduced a bug. Will investigate immediately.

**Comment by dodsky** [comment]
@seblau seems to be working fine on:
node 7.2.0 / npm 4.6.1 on mac
node 8.5.0 / npm 5.4.2 on mac
node 7.2.1 / npm 5.3.0 on win

Seems to be connected to some git related npm issue. We've released a version with fixed versions - 0.9.9. Can you try again?

**Comment by Vladimir-Vdovic** [comment]
We looked into it, the source of the issue is a dependency in one of your custom extensions.

We believe it’s:
```
"react-native-beacons-manager": "https://github.com/peterdeka/react-native-beacons-manager.git#b3e12d5343e9272bf994128dee5114eeeb35d557",
```

Since the error you receive states:
`fatal: reference is not a tree: b3e12d5343e9272bf994128dee5114eeeb35d557`

That specific commit seems to no longer exist.

Closing issue.
